### ISSUE TITLE
Genesis Mini: Have your Game List based on the language selected (as an option)

### ISSUE THREAD

**Issue by ReyVGM** [TeamShinkansen/Hakchi2-CE#276]
Right now, with a hacked system, if you change the language of the Mini, your game list doesn't switch to the games released for that language; as opposed to an unhacked original system that gives you a modified list of games based on the what was released for that region. For example, Dr. Robotnik's Mean Bean Machine changes to Puyo Puyo when the language is set to Japanese. Lunar already keeps that behavior, but Hakchi doesn't have it.

My request: Add this option for the people that want it. If someone wants to have US/Japanese/Euro variants of the same game to only appear if the language is changed, then I believe the option should be there just like the original console has it. Again, it's optional and it helps the Mini feel more authentic for the people that want it.

**Comment by cadasag** [comment]
I agree, this is the only feature missing that would make it perfect for me. Sadly I don't think this is an option on Project Lunar when importing new games, either.

**Comment by ReyVGM** [comment]
Maybe one day it will happen!
